helper/pluginutil/version_test.go

```diff
@@ -0,0 +1,57 @@
+package pluginutil
+
+import (
+	"os"
+	"testing"
+)
+
+func TestGRPCSupport(t *testing.T) {
+	cases := []struct {
+		envVersion string
+		expected   bool
+	}{
+		{
+			"0.8.3",
+			false,
+		},
+		{
+			"0.9.2",
+			true,
+		},
+		{
+			"0.9.2+ent",
+			true,
+		},
+		{
+			"0.9.2-beta",
+			false,
+		},
+		{
+			"0.9.3",
+			true,
+		},
+		{
+			"unknown",
+			true,
+		},
+		{
+			"",
```

Comment:
This shows up somewhat weird on the test output, wondering if it's worth adding a `name` field for this
```
=== RUN   TestGRPCSupport
=== RUN   TestGRPCSupport/0.8.3
=== RUN   TestGRPCSupport/0.9.2
=== RUN   TestGRPCSupport/0.9.2+ent
=== RUN   TestGRPCSupport/0.9.2-beta
=== RUN   TestGRPCSupport/unknown
=== RUN   TestGRPCSupport/#00
--- PASS: TestGRPCSupport (0.00s)
    --- PASS: TestGRPCSupport/0.8.3 (0.00s)
    --- PASS: TestGRPCSupport/0.9.2 (0.00s)
    --- PASS: TestGRPCSupport/0.9.2+ent (0.00s)
    --- PASS: TestGRPCSupport/0.9.2-beta (0.00s)
    --- PASS: TestGRPCSupport/unknown (0.00s)
    --- PASS: TestGRPCSupport/#00 (0.00s)
PASS
ok  	github.com/hashicorp/vault/helper/pluginutil	0.014s
```